app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments, foreign_key: :post_id
   has_many :likes, foreign_key: :post_id
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def recent_comments
     comments.order(created_at: :desc).limit(5)
   end
```

Comment:
- [ ] Kindly test this function and the one that __updates the posts counter__ as it is a requirement to test all the model methods.